akka-docs/src/main/paradox/logging.md

```diff
@@ -400,6 +406,26 @@ a non-blocking appender. Logback provides [AsyncAppender](http://logback.qos.ch/
 that does this. It also contains a feature which will drop `INFO` and `DEBUG` messages if the logging
 load is high.
 
+### Logback configuration
+
+Logback has flexible configuration options and details can be found in the
+[Logback manual](https://logback.qos.ch/manual/configuration.html) and other external resources.
+
+One part that is important to highlight is the importance of configuring an [AsyncAppender](http://logback.qos.ch/manual/appenders.html#AsyncAppender).
```

Comment:
```suggestion
One part that is important to highlight is the importance of configuring an [AsyncAppender](http://logback.qos.ch/manual/appenders.html#AsyncAppender), because it offloads rendering of logging events to a background thread, increasing performance.  It also contains a feature which will drop `INFO` and `DEBUG` messages if the logging load is high.
```

Also maybe mention https://github.com/logstash/logstash-logback-encoder#async-appenders which uses LMAX disruptor?